website/templates/emails/pending_registration_admin.txt.mako

```diff
@@ -0,0 +1,16 @@
+Hello ${user.fullname},
+
+
+We just wanted to let you know that ${initiated_by} has initiated the following pending registration: ${registration_link}
+
+To approve this action click the following link: ${approval_link}
+
+To disapprove this action, click the following link: ${disapproval_link}
+
+Note: Clicking the disapproval link will immediately cancel the pending registration. If you neither approve nor disapprove
+the registration within ${approval_time_span} hours of midnight tonight (EDT) the registration will become automatically
+approved. This operation is irreversible.
+
+Sincerely yours,
+
+The OSF Robots
```

Comment:
By the way, "Let's not decide right now" and "It just doesn't matter" are both acceptable responses to my question.